src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -65,6 +66,7 @@ internal class GrpcWorker : IWorker
             _serializer = workerOptions.Value.Serializer ?? throw new InvalidOperationException(nameof(workerOptions.Value.Serializer));
             _inputConversionFeatureProvider = inputConversionFeatureProvider ?? throw new ArgumentNullException(nameof(inputConversionFeatureProvider));
             _functionMetadataProvider = functionMetadataProvider ?? throw new ArgumentNullException(nameof(functionMetadataProvider));
+            _invocationCancelDictionary = new Dictionary<string, CancellationTokenSource>();
```

Comment:
I just put this here to get myself unblocked, but again looking for guidance on the right approach here. We need to have a cancellation token source associated with each invocation (via invocationId), and since the context is created in `InvocationRequestHandlerAsync` we do not have access to the cancellation token or cts at all. 

@fabiocav mentioned maybe keeping track of invocations and their context somewhere (instead of the invocationId and cts like I am doing here) - any thoughts on this?